noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -945,40 +1419,15 @@
                 ]]>
             </errors>
         </queries>
-    </rule>
-
-    <rule>
-        <title>Files within classes</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT f._ref_arkivdel series_system_id, f._ref_klasse class_system_id, COUNT(*) number_of_files
-                    FROM arkivstruktur.mappe f
-                    GROUP BY f._ref_arkivdel, f._ref_klasse;
-                ]]>
-            </info>
-        </queries>
-    </rule>
-
-    <rule>
-        <title>Files without subfolders or records.</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT f._ref_arkivdel series_system_id, COUNT(*) empty_file_count
-                    FROM arkivstruktur.mappe f
-                    WHERE f.systemid NOT IN (SELECT _ref_mappe FROM arkivstruktur.mappe WHERE _ref_mappe IS NOT NULL GROUP BY _ref_mappe)
-                        AND f.systemid NOT IN (SELECT _ref_mappe FROM arkivstruktur.registrering WHERE _ref_mappe IS NOT NULL GROUP BY _ref_mappe)
-                    GROUP BY f._ref_arkivdel;
-                ]]>
-            </info>
-        </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST9">
         <title>File status</title>
+        <description>
+            Tests whether a non-finalized File exists in the archive.
```

Comment:
`Tests whether non-finalized files exists in the archive. A file is considered non-finalized if a finalized date or a finalizing party is not specified, or if the file status is not 'Avsluttet'`